packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -1223,6 +1227,9 @@ Future<OperationResult> _defaultReloadSourcesHelper(
     // Reset devFS lastCompileTime to ensure the file will still be marked
     // as dirty on subsequent reloads.
     _resetDevFSCompileTime(flutterDevices);
+    if (reloadReport == null) {
+      return OperationResult(1, 'No Dart isolates found');
```

Comment:
Is this right, or should we toolexit?

Reply:
So...uh...hmm

So, we have to have at least some isolates running, since the engine must be running and so does the service isolate. But we are filtering those out - maybe we're filtering out other isolates too?

From the attached issue, it looks like the user is running the command with the added arg "--app-id com.klook.klook". Is it possible that this app-id string is further filtering down the list of isolates (because there is some mismatch), resulting in none being found?

Just spitballing here. I think the change itself is fine.

Reply:
Yeah good point. And if `--app-id` is wrong, how'd we find the app to attach to? I have no idea what's going on here.